lib/utils/__tests__/validateOptions.test.js

```diff
@@ -143,6 +143,18 @@ describe("validateOptions for secondary options objects", () => {
       'Invalid option value 2 for rule "foo": should be an object'
     );
   });
+
+  it(`should have "possible" if "optional" is set`, () => {
+    validateOptions(result, "foo", {
+      optional: true,
+      actual: { foo: "always" }
+    });
+
+    expect(result.warn.mock.calls[0]).toHaveLength(2);
+    expect(result.warn.mock.calls[0][0]).toEqual(
```

Comment:
Can you use `toHaveBeenCalledTimes` and `toHaveBeenCalledWith`? Just easier to read for someone unfamiliar with Jest.

Reply:
I used same matchers as other tests in this file for consistency.

Reply:
Weird, I thought I changes all of those when we removed Sinon. Oh well, point taken.